ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -114,7 +122,19 @@ OnboardingBasePage {
                 }
             }
 
+            function checkPasswordMatches() {
+                if (confPswInput.text !== root.password) {
+                    errorTxt.text = qsTr("Passwords don't match")
+                    return false
+                }
+                return true
```

Comment:
You probably need to reset the `errorTxt.text` to empty here.

Reply:
maybe it should be empty when password length changed... but I don't think about `validators` cos here should be just one - password length.